app/user/user-resolver.js

```diff
@@ -1,12 +1,11 @@
 const config = require('config');
 const fetch = require('../util/fetch');
+const jwtUtil = require('../util/jwt');
 
 const getUserDetails = (jwt) => {
-  let bearerJwt = jwt.startsWith('Bearer ') ? jwt : 'Bearer ' + jwt;
-
   return fetch(`${config.get('idam.base_url')}/o/userinfo`, {
```

Comment:
jwt can be undefined .  ! in some cases .

Reply:
That line has just been moved, no actual code/logic change in regards to this on this PR 👍